Makefile

```diff
@@ -14,22 +14,25 @@ subtree-preflight:
 	git gc
 .PHONY: subtree-preflight
 
+PULL_PREFIX ?=
+PUSH_PREFIX ?= $(USER)/from-telepresence.io-$(shell date +%Y-%m-%d)/
+
+dir2branch = $(patsubst docs/%,release/%,$(subst pre-release,v2,$1))
+
 pull-docs: ## Update ./docs from https://github.com/datawire/ambassador-docs
```

Comment:
this comment is incorrect right since we're not pushing to `datawire/ambassador-docs` anymore right? And are instead pushing to telepresenceio/docs?

Reply:
Indeed.